docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
```

Comment:
Nitpicking: this bullet point ends in a semicolon and all the others in a full stop.

Reply:
As per convention, all but the final sentence in the bullet points, ends with `;`.